src/api/Interpret.cc

```diff
@@ -129,10 +134,9 @@ void Interpret::exit() {
     f_exit = true;
 }
 
-using opensmt::Logic_t;
-using namespace osmttokens;
+using namespace tokens;
 
-namespace {
+namespace aux {
```

Comment:
It does not compile without it :) because now it would be an ambiguous call. Previously, the original impl. was in opensmt namespace and the anonymous not, but now both are in opensmt namespace and `getLogic` would be ambiguous and `opensmt::getLogic` would not call the unnamed namespace , and it is not possible to call it. Should I use anonymous namespace but alter the name of the function, e.g. by adding `_aux`?

Reply:
I see. Just leave it as it is now :)

Reply:
I kept the anonymous namespace and used a different identifier instead to keep the linkage internal.